### PR TITLE
announcement API fixes

### PR DESCRIPTION
**Pull request by therealmarv**

### CODE REVIEW

File: alembic/versions/52209719b79e_extend_announcements_with_media_and_.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return now.isoformat()
+
+
+def upgrade():
+    op.add_column('announcement', sa.Column(field, sa.String))
+    op.add_column('announcement', sa.Column('info', JSON))
+    op.add_column('announcement', sa.Column('published', sa.Boolean, default=False))
+    op.add_column('announcement', sa.Column('updated', sa.Text,
+                                       default=make_timestamp))
+    sql = 'update announcement set published=true'
```
Comment (teleyinex) [inline alembic/versions/52209719b79e_extend_announcements_with_media_and_.py]: You are missing here media_url

Comment (therealmarv) [reply alembic/versions/52209719b79e_extend_announcements_with_media_and_.py]: it's there, only not good visible. Changed that in https://github.com/Scifabric/pybossa/pull/1644/commits/f2ff63cb6de024e5336587a1bd8ad8c2a12eb32b

File: test/test_api/test_api_announcement.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        url = '/api/announcement'
+        res = self.app_get_json(url)
+        data = json.loads(res.data)
+        assert len(data) == 10, data
+
+        # As user
+        res = self.app_get_json(url + '?api_key=' + user.api_key)
+        data = json.loads(res.data)
+        # TODO: project_id is tried to load with api_key. Do not load it?
+        assert data['exception_cls'] == 'AttributeError', data
```
Comment (teleyinex) [inline test/test_api/test_api_announcement.py]: This should return all the announcements, correct?

Comment (teleyinex) [reply test/test_api/test_api_announcement.py]: Basically, you need to remove the owner_id from the context manager to ensure this works.

**Comment by teleyinex** [comment]
:cake: Fix the conflicts :D

**Comment by therealmarv** [comment]
really coveralls? wtf it is telling everywhere increasing and at the end not?

**Pull request merged**